jovian/cli.py

```diff
@@ -145,6 +146,17 @@ def set_project(ctx,  notebook, project):
     set_notebook_slug(filename=notebook, slug=project)
 
 
+@main.command("commit", short_help="Create a new notebook on Jovian")
+@click.argument('notebook')
+@click.pass_context
+def exec_commit(ctx,  notebook):
+    """Create a new notebook on Jovian
+
+        $ jovian commit my_notebook.ipynb
+    """
+    commit(filename=notebook, environment=None, is_cli=True)
```

Comment:
This will break support for Python 2.x

Reply:
Can we put this behind a Python version check?

If user tries to run this from Python 2.x, then we can print a warning "committing is not supported from Python 2.x. Please install & run jovian from Python 3.5 or above"

Reply:
Appears to be working fine. I've added a warning message with soft check for 2.x.